src/averageAges.js

```diff
@@ -15,7 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const menByCentury = people
+    .filter(el => el.sex === 'm' && Math.ceil(el.died / 100) === century);
+
+  const men = people
+    .filter(el => el.sex === 'm');
```

Comment:
1. You can reuse your men array to avoid repeating the same condition
2. Use decomposition as much as possible
3. If century is `false` then you won't use menByCentury but you calculate it anyway. Try to make `if` conditions and create `menByCentury  `variable only in case `century === true`
```suggestion
  const men = people.filter(({ sex }) => sex === 'm');
    
  const menByCentury = men.filter(({ died }) => Math.ceil(died / 100) === century); 
```